### PR TITLE
setup spring-cloud as parent instead of spring-boot, and add spring-b…

### PR DESCRIPTION
**Pull request by jhulick**
After reading this post on the [Brixton](https://spring.io/blog/2015/09/16/first-milestone-of-spring-cloud-brixton-release-train-is-available) release, I think it would be better to follow their guide on pom.xml setup for spring cloud. The idea I'm suggesting is to manage the dependencies via Spring Cloud [Release Train](http://projects.spring.io/spring-cloud/) (see bottom page of link) so that the currated sets of dependencies are aligned. Sorry I did not research this fully before providing PR #19.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,38 +12,30 @@
...
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-config-server</artifactId>
-            <version>${spring-cloud.version}</version>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+
+        <dependency>
```
Comment (jdubois) [inline pom.xml]: Why are you adding this, is it for Gradle support?

Comment (jhulick) [inline pom.xml]: @jdubois I've added it to manage spring cloud dependencies as recommended in the [Eureka Server Setup](http://projects.spring.io/spring-cloud/spring-cloud.html#spring-cloud-eureka-server). Though it's show in the docs as an Gradle example, the plugin works with Maven too.

**Comment by jdubois** [comment]
OK we can try this. As long as this doesn't force us to have an old Spring Boot version.

**Comment by jhulick** [comment]
@jdubois The current Brixton.BUILD-SNAPSHOT seems to have the latest snapshot of each library/framework in its collection. Thanks.

**Pull request merged**